modules/values-default.yaml

```diff
@@ -27,6 +27,7 @@ namespaceConfiguratorEnabled: true
 nodeManagerEnabled: true
 operatorPrometheusCrdEnabled: true
 operatorPrometheusEnabled: true
+operatorTrivyEnabled: true
```

Comment:
NIT I think trivy operator should be false by default

Reply:
Done.